### ISSUE TITLE
ResNet50 in Eager mode. Model training set to True and False

### ISSUE THREAD

**Issue by mhamdan91** [tensorflow/tensorflow#26414]
I am using the Resnet50 code maintained under Eager Execution as is. I am using the following code for training: **_prediction = model(x, training=True)_**
When I try to validate while training, i replace **training=True** with **training=False**. This however, results in a significant change in the loss value as if the model has not been trained at all. 
Can you please explain to me what is going on, and how to correct this issue?

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new/choose). Thanks!
